wger/core/models.py

```diff
@@ -647,3 +646,9 @@ def is_weight(self):
         This is done basically to not litter the code with magic IDs
         '''
         return self.id in (1, 2)
+
+
+# Model to show users created via api
+class ApiUsers(models.Model):
+    app_owner = models.ForeignKey(User, related_name='app_owner')
+    app_user = models.ForeignKey(User, related_name='app_user')
```

Comment:
should this be `app_` or `api_`

Reply:
App because the api owner is wger which allows an app creator to use the api. The app_user is the user created by the app creator or app_owner.